idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -577,4 +577,61 @@ service WorkflowService {
       6: shared.ClientVersionNotSupportedError clientVersionNotSupportedError,
     )
 
+  /**
+  * StartBatchJob starts a new batch job
+  **/
+  shared.StartBatchJobResponse StartBatchJob(1: shared.StartBatchJobRequest startRequest)
+    throws (
+      1: shared.BadRequestError badRequestError,
+      2: shared.InternalServiceError internalServiceError,
+      3: shared.WorkflowExecutionAlreadyStartedError sessionAlreadyExistError,
```

Comment:
What workflow execution has to do with batch?

Reply:
Removed.